### DISCUSSION TITLE
DIDs help

### DISCUSSION THREAD

**Discussion by Ozodbek1997** [decentralized-identity/veramo#759] [Q&A]
Hello everyone,
I am new here and do not have much knowledge about DIDs.
My thesis subject is in DIDs.
Currently, I am reading papers and posts about DIDs.But I feel not confident with DIDs.

Can you please give me direction or advice to be a Master in DIDs that I will be able to do some projects using DIDs ?
Please help me!

**Comment by mirceanis** [comment]
Hi, perhaps these links will also help you in your quest:

https://identity.foundation/faq/#introductory-concepts-layer-0

https://w3c-ccg.github.io/did-primer/

It's hard to tell from your question how far you've come along.
Try to ask for something more specific to get a better answer.
